### PR TITLE
Updating sesame component docs

### PR DESCRIPTION
**Pull request by zanglang**
**Description:**

Updating the sesame component's documentation with the api_key config needed for V3 API.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23621

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sesame.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: page
 title: "Sesame Smart Lock"
 description: "Instructions on how to integrate Sesame by CANDY HOUSE into Home Assistant."
-date: 2017-05-02 12:00
+date: 2019-04-30 12:00
```
Comment (klaasnicolaas) [inline source/_components/sesame.markdown]: Please do not change this

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/sesame.markdown (hunk 1 above)
Comment (zanglang) [reply source/_components/sesame.markdown]: My bad, thanks!

**Comment by zanglang** [comment]
Hi, any chance we could get this merged? This PR adds a URL for sesame users to generate an API key, which is a breaking change just released in home-assistant.

**Review by robbiet480 — APPROVED** [review]

**Comment by robbiet480** [comment]
@zanglang ask and ye shall receive

**Pull request merged**